### DISCUSSION TITLE
Spaces defined on different meshes.

### DISCUSSION THREAD

**Discussion by enricofacca** [firedrakeproject/firedrake#2859] [Firedrake support]
Is it possible to work with spaces defined on different meshes, where the one mesh is the refinement of the other? 

The following MWE leads to the error
 "TypeError: '<' not supported between instances of 'MeshGeometry' and 'MeshGeometry' "

`from firedrake import *

mesh_triangle = UnitSquareMesh(10,10)
meshes = MeshHierarchy(mesh_triangle, 2)
mesh_coarse = meshes[0]
mesh_fine = meshes[1]

V = FunctionSpace(mesh_coarse, "DG", 0)
W = FunctionSpace(mesh_fine, "CG", 1)

x, y = SpatialCoordinate(mesh_fine)
pot = Function(W).interpolate(x)

test = TestFunction(V)
F = assemble(test * inner(grad(pot),grad(pot))*dx)
`
Enrico

**Comment by wence-** [comment]
It is not possible at present, sorry.
